multiply.py

```diff
@@ -0,0 +1,11 @@
+"""
+This is an example docstring.
+
+A docstring with a summary line, a blank line
+and some more text like here.
+"""
```

Comment:
I'd suggest not just copy-pasting docstrings for the sake of having them.

It's a bit of an art to write good docstrings, and in general the more practice you have at deliberately / intentionally making good ones, the better, because you'll develop more of an intuition then as a result of trying more things.